sdk/src/client/bindings/python/iota_client/client.py

```diff
@@ -857,3 +919,25 @@ def as_dict(self):
             config['maximum_supply'] = str(hex(config['maximum_supply']))
 
         return config
+
+class Output():
```

Comment:
I think we want to use this name for an actual output type later, so maybe call it `AddressWithAmount` as we do in the Rust lib

Reply:
Yeah makes sense. Thought about keeping it simple but knew about the danger of you wanting to use it in the future.
42 Gi and you can have the name :trollface:

Reply:
Greedy, 1.2 Gi or I will close this PR :trollface:

Reply:
Deal and done